api/base/types.go

```diff
@@ -36,6 +36,7 @@ type ModelStatus struct {
 	HostedMachineCount int
 	ApplicationCount   int
```

Comment:
At the moment `len(r.Applications)` != `r.ApplicationCount` because of the controller application. This could be confusing